python/nammu/view/ConsoleView.py

```diff
@@ -44,17 +48,30 @@ def __init__(self, controller):
         self.setLayout(BorderLayout())
 
         # Create console-looking area
-        self.edit_area = JTextArea()
-        self.edit_area.setLineWrap(True)
-        self.edit_area.setWrapStyleWord(True)
```

Comment:
Wrap settings not needed for JEditorPane? I think from what I tried it seemed ok, but just double checking.